parsexml.py

```diff
@@ -27,11 +27,14 @@ def processFile(fname):
 
             #get document id from metadata xml file
             for x in doc['main']['DATA_RECORD']:
+                x['appid'] = x.pop('BD_PATENT_APPLICATION_NO')
+                x['doc_date'] = x.pop('DOCUMENT_CREATE_DT')
```

Comment:
I did not see Art unit or anything resembling it in the xml schema of the metadata file.